docker/Dockerfile

```diff
@@ -51,7 +51,12 @@ WORKDIR /dav1d/build
 
 RUN meson setup ../; \ 
     ninja; \
-    ninja install 
+    ninja install; \
+    ln -s /usr/local/lib/x86_64-linux-gnu/libdav1d.so.7.0.0 /usr/local/lib/libdav1d.so.7; \
+    ln -s /usr/local/lib/x86_64-linux-gnu/libdav1d.so.7.0.0 /usr/local/lib/libdav1d.so.7.0; \
+    ln -s /usr/local/lib/x86_64-linux-gnu/libdav1d.so.7.0.0 /usr/local/lib/libdav1d.so.7.0.0; \
+    echo "/usr/local/lib" >> /etc/ld.so.conf.d/mylibs.conf \
+    && ldconfig
```

Comment:
This could probably be done a lot more elegantly and generically but this seems to work so I'd be interested to see how I could do it better

Reply:
I won't have time to test this until after work, but this won't account for arm64 builds

Reply:
Yeah I'm happy to look at it again tomorrow to work out how to make it more generic (like using `TARGETARCH` or something). Defs just a hack for now to get it to work for me
If you have an idea of how to improve I'll get to it in the morning but it's late over here so gonna call it for this attempt